src/components/Home.js

```diff
@@ -106,15 +132,49 @@ export default function Home() {
       </div>
 
       <section className={'mt3 section partners'}>
-        <h1 className={'title has-text-centered'}>Partners</h1>
+        <h1 className={'title has-text-centered is-1'}>Partners</h1>
         <div className={'container is-flex'}>
           {partners.map((partner, index) => (
-            <figure key={index} className={'image'}>
-              <img src={partner} alt={''} />
-            </figure>
+            <a href={partner.url} className={'border-shadow'}>
+              <figure key={index} className={`image is-full-height ${index % 2 === 0 ? 'is-flex' : ''}`}>
+                <img src={partner.logo} alt={partner.name} />
+              </figure>
+            </a>
           ))}
         </div>
       </section>
+
+      {/* <section className={'section'}>
+        Team???
+      </section> */}
+
+      <section className={'mt3 section has-background-color-darker'}>
+        <div className={'container'}>
+          <h1 className={'title is-1'}>Contribute</h1>
+          <h2 className={'subtitle has-75-percent-width'}>
+            The project is made by engineers. Everything being open source, you can contribute directly to it by
+            providing documentation, translating it, or directly coding new features. You can also donate directly
+            towards the project.
+          </h2>
+          <div className={'buttons'}>
+            <a
+              href={'https://github.com/blockchainworkbench'}
```

Comment:
adding `target={'_blank'} rel={'noopener noreferrer'}` here like on external links in footer would make sense

Reply:
fixed for github and donation url